src/test/resources/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
     image: symptoma/activemq
 
   communication:
-    image: repo.backbase.com/docker-releases/communication:2022.11
+    image: harbor.backbase.eu/staging/communication:2024.08
```

Comment:
Hi @afsinka-backbase, Backbase organisation (public repository) always points to public repo and not internal Harbor, please keep:

```
repo.backbase.com/docker-releases/communication:2024.08
```